spec/features/idv/steps/review_step_spec.rb

```diff
@@ -0,0 +1,113 @@
+require 'rails_helper'
+
+feature 'idv review step', :idv_job do
+  include IdvStepHelper
+
+  it 'requires the user to enter the correct password to redirect to confirmation step' do
+    start_idv_from_sp
+    complete_idv_steps_before_review_step
+
+    fill_in 'Password', with: 'this is not the right password'
+    click_idv_continue
+
+    expect(page).to have_content(t('idv.errors.incorrect_password'))
+    expect(page).to have_current_path(verify_review_path)
+
+    fill_in 'Password', with: user_password
+    click_idv_continue
+
+    expect(page).to have_content(t('headings.personal_key'))
+    expect(page).to have_current_path(verify_confirmations_path)
+  end
+
+  it 'displays the profile information for the user' do
+    start_idv_from_sp
+    complete_idv_steps_before_review_step
+
+    expect(page).to have_content('José')
+    expect(page).to have_content('One')
+    expect(page).to have_content('123 Main St')
+    expect(page).to have_content('Nowhere, VA 6604')
+    expect(page).to have_content('January 02, 1980')
+    expect(page).to have_content('666-66-1234')
+    expect(page).to have_content('+1 (555) 555-0000')
+  end
```

Comment:
Can we combine this `it` block into the previous one since all this is doing is expecting some info to be displayed before filling in the password in the previous spec. Compounded, all of these expectation consolidations will have a significant impact on the speed of the test suite.